pom.xml

```diff
@@ -1528,6 +1824,55 @@
                                     org.wso2.carbon.deployment:org.wso2.carbon.webapp-runtime-environment.category.feature:${carbon.deployment.version.4.5.3}
                                 </featureArtifactDef>
 
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.agents.category.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.authorization.category.feature:${carbon.identity.version.5.0.7}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.identity:org.wso2.carbon.identity.authenticators.category.feature:${carbon.identity.version.5.0.7}
```

Comment:
org.wso2.carbon.identity.authorization.category.feature and org.wso2.carbon.identity.authenticators.category.feature are out of order